rails/app/controllers/api/v1/report_learners_es_controller.rb

```diff
@@ -1,5 +1,9 @@
+require 'digest/md5'
+
 class API::V1::ReportLearnersEsController < API::APIController
 
+  PAGINATED_ELASTIC_SEARCH_SIZE = 1000
```

Comment:
@scytacki Wondering whether this should be part of the external report's request to the portal. That would allow us to more easily update this number, since it would only require updating the report app.

We could still have an internal cap, and reject the request if it asks for something higher (rather than send down something with the wrong number of leaners per page).

Reply:
Yes that is a good idea. It is a pretty standard way paging apis work.